src/ripple/rpc/handlers/WalletPropose.cpp

```diff
@@ -26,13 +27,18 @@ namespace ripple {
 // }
 Json::Value doWalletPropose (RPC::Context& context)
 {
+    return WalletPropose (context.params);
+}
+
+Json::Value WalletPropose (Json::Value const& params)
```

Comment:
It'd be really nice not to have this function start with an uppercase letter... almost all our functions are camelCase and the rest use_underscores...